lib/BC/InstructionLifter.cpp

```diff
@@ -290,9 +291,25 @@ InstructionLifter::LoadRegAddress(llvm::BasicBlock *block,
     reg_ptr_it->second = reg_ptr;
     return reg_ptr;
 
+  // Try to find it as a global variable.
+  } else if (auto gvar = module->getGlobalVariable(reg_name)) {
+    return gvar;
+
+  // Invent a fake one and keep going.
   } else {
-    LOG(FATAL) << "Could not locate variable or register " << reg_name_;
-    return nullptr;
+    std::stringstream unk_var;
+    unk_var << "__remill_unknown_register_" << reg_name;
+    auto unk_var_name = unk_var.str();
+    auto var = module->getGlobalVariable(unk_var_name);
+    if (!var) {
+      LOG(ERROR)
+          << "Could not locate variable or register " << reg_name_;
```

Comment:
`LOG(ERROR)` is problematic as user cannot do anything about it (method returns nothing about it) and user cannot configure the stream. We will need to fix them on whole project level anyway though, but if this is not something we want to propagate up to the user eventually, I would just remove it.

Reply:
This can be simplified right?

```
if (auto var = module->getGlobalVariable(unk_var_name)) {
  return var;
}
return new llvm::GlobalVariable( ... )
```

Reply:
Add a comment then that will give us a hint what should be done once `LOG` is refactored?